drivers/gpio/gpio_b9x.c

```diff
@@ -676,12 +700,15 @@ static int gpio_b9x_pm_action(const struct device *dev, enum pm_device_action ac
 			data->gpio_b9x_retention.analog_in_conf
 			= analog_read_reg8(areg_gpio_pd_ie);
 		}
+#if CONFIG_SOC_RISCV_TELINK_B91
 		if (IS_PORT_F(gpio)) {
```

Comment:
Else if seems more clear.
For B91 there is no issue with pull resistor?

Reply:
Shanghai team didn't reported the issues for B91 MCU.